Benchmarking Long-Read Aligners and SV Callers for Structural Variation Detection in Oxford Nanopore Sequencing Data: A Comprehensive Evaluation
​
## For GM24385 Dataset
​
```bash
truvari -f ref/human_hs37d5.fasta -b giab/HG002_SVs_Tier1_v0.6.vcf.gz --
includebed giab/HG002_SVs_Tier1_v0.6.bed -o bench-SVcaller --passonly -p
0.00 -c tools/SVcaller/SVcaller.vcf.gz
```
​
## For SI00001 Simulated Dataset
The benchmark SV set generated according to (Bolognini and Magi, 2021)
​
```bash
truvari -f ref/human_hs37d5.fasta -b benchmark/hack.vcf.gz --includebed
benchmark/hack.bed -o bench-SVcaller --passonly -p 0.00 -c
tools/SVcaller/SVcaller.vcf.gz
```
​
## For NA12878 Dataset
The benchmark SV set used for comparison was the SV vcf generated by Nanopore-wgs-consortium (Jain et al., 2018) using bed file extracted from the benchmark set VCF
​
```bash
truvari -f ref/human_hs37d5.fasta -b benchmark/ NA12878.hq.sv.vcf.gz --
includebed benchmark/NA12878.hq.sv.bed -o bench-SVcaller --passonly -p
0.00 -c tools/SVcaller/SVcaller.vcf.gz
```
